packages/botonic-plugin-google-analytics-v4/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@botonic/plugin-google-analytics",
+  "version": "0.20.0",
```

Comment:
Change: The version for this package should be `0.22.0`, right? The latest one released is the `0.21.0`.

Reply:
changed 👌